starter-code/src/components/CountryDetail.js

```diff
@@ -0,0 +1,46 @@
+import React, {useContext} from 'react';
+
+import { CountryContext } from '../contexts/CountryContext';
+import { Link } from 'react-router-dom';
+
+
+const CountryDetail = (props) => {
+  const {findCountry} = useContext(CountryContext);
+  const cca3 = props.match.params.cca3;
+  const country = findCountry(cca3);
```

Comment:
Aquí podéis utilizar `object destructuring` para que el manejo de las variables sea más cómodo después y no tengáis que estar escribiendo `country....`
```
const { cca3 } = props.match.params;
const { name, capital, area, borders } = findCountry(cca3);
```